foam_controlwrapper/examples/foam_mesh_to_h5cuds.py

```diff
@@ -11,5 +11,4 @@
 path = '.'
 mesh_inside_wrapper = read_foammesh(name, path)
 mesh_file = H5CUDS.open("poiseuille.cuds")
```

Comment:
maybe this example should be renamed to `foam_mesh_to_simphony_mesh.py` and should stop at line 12.